src/metrics/blocks.mli

```diff
@@ -0,0 +1,3 @@
+val inc_block_produced : unit -> unit
```

Comment:
If we keep using a API like this we can be sure we're not exposing anything we shouldn't by just looking in the `metrics` library